sorting/go/bubble-sort.go

```diff
@@ -0,0 +1,73 @@
+package main
```

Comment:
You should also add a little info about the algo itself like time complexity and a little explanation.